model.py

```diff
@@ -7818,6 +7890,16 @@ class RightsStatus(Base):
         GENERIC_OPEN_ACCESS,
     ]
 
+    ALLOWS_DERIVATIVES = [
```

Comment:
A comment would be useful here for the future. I'm also not convinced that GENERIC_OPEN_ACCESS should be in this list. That's often a dumping ground for 'you can have a copy of this but there's been no formal relicensing'.